### DISCUSSION TITLE
reset using remote

### DISCUSSION THREAD

**Discussion by biau** [Koenkk/zigbee2mqtt#5502] [Devices]
Hi All,

I've a float panel like this:
https://www.zigbee2mqtt.io/devices/L1528.html#ikea-l1528
And a remote like this:
https://www.zigbee2mqtt.io/devices/E1743.html#ikea-e1743

I have no physical switch on the float panel so I would like to know if there's a way to reset the panel using the remote and make it join my z2m network ?
It's actually in my network but I would like to change my actual adapter CC2531 and try using zigate.

Thanks

Stéphane

**Comment by pcb1962** [comment]
To reset IKEA devices switch them on and off six times.

**Reply by biau** [reply]
yes I know that but thanks.
But the float panel does not have physical switch anymore, so I have to disconnect the power cable and add a switch on it for the reset.
I was wondering if I touchlink the float pannel with a remote already on the z2m network if the panel will join the same network (like explained in the ikea video here https://www.youtube.com/watch?v=5yzhOy1THPg )

**Comment by biau** [comment]
I've seen on different web site that resetting a bulb is possible with a hue dimmer by pressing on and off button at the same time for 10s using touchlink.
I did not find any way to reproduce this with my small ikea remote
